.travis.yml

```diff
@@ -8,6 +8,8 @@ script:
 - echo "Nginx Prometheus Exporter - commit:${TRAVIS_COMMIT}"
 - make test
 - make container
+- make release && echo "Nginx Prometheus exporter binaries built:" && ls -alh build/nginx-prometheus-exporter* && cat build/sha256sums.txt
+- make clean
```

Comment:
why do we need to clean?

Reply:
In our travis pipeline we will push the artifacts to wherever we need and then clean the environment. No relevant now, but as soon as we have the complete release, I would like to clean the environment. I know it is a container but in other cases this could be a vm or local environments.